test/integration/integration_test.go

```diff
@@ -112,7 +114,7 @@ func TestMain(m *testing.M) {
 func parseFlags() {
 	flag.StringVar(&minishiftArgs, "minishift-args", "", "Arguments to pass to minishift")
 	flag.StringVar(&minishiftBinary, "binary", "", "Path to minishift binary")
-
+	flag.StringVar(&execBeforeFeature, "exec-before-feature", "", "Set of minishift commands to be executed before every feature. Individual commands must be delimited by a semicolon.")
```

Comment:
I would say put that delimiter as const variable and then use this.

Reply:
Also show what the delimiter is, else I would likely try `:`... user thinking "what's a semi-colon again?", before realizing it might be `;` ;-P